ufl/finiteelement/finiteelement.py

```diff
@@ -236,3 +237,9 @@ def __getnewargs__(self):
                 None,
                 self.quadrature_scheme(),
                 self.variant())
+
+    def is_fully_continuous(self):
+        """Return true if the values of this element's basis functions are continuous between elements."""
+        if self.family() == "Real":
+            return True
+        return self in H1
```

Comment:
Probably "Real" finite elements should report that they are in $C^\infty$ and then this special-casing should not be necessary.

Reply:
I agree, but as there's no Cinfinity space in sobolevspace.py yet, I think this might be one for a future PR. I'll open an issue for it.

Reply:
#114

Reply:
You could get rid of this special case by just shifting "Real" into "H2", which is more accurate than the current situation with it being in "L2".